evalml/tests/component_tests/test_time_series_regularizer.py

```diff
@@ -325,3 +325,36 @@ def test_ts_regularizer_uneven(
 
     error_dict = ts_regularizer.error_dict
     assert_features_and_length_equal(X, y, X_output, y_output, error_dict)
+
+
+@pytest.mark.parametrize(
+    "missing",
+    [[], [10], [10, 11], [10, 11, 15]],
+)
+def test_ts_regularizer_multiseries(multiseries_ts_data_unstacked, missing):
```

Comment:
I think we need more robust testing here - there are more scenarios that will need the regularizer than just missing values. Can you parametrize the existing tests to cover the multiseries case?